velox/dwio/parquet/reader/ParquetReader.cpp

```diff
@@ -76,9 +107,7 @@ void toDuckDbFilter(
     case common::FilterKind::kBoolValue:
     case common::FilterKind::kBigintValuesUsingHashTable:
     case common::FilterKind::kBigintValuesUsingBitmask:
-    case common::FilterKind::kDoubleRange:
     case common::FilterKind::kFloatRange:
```

Comment:
Why did you leave float range here and remove bytes range ? I would expect the float and double logic to be the same (minus the type casts that is).

Reply:
Simply because I do not have data to test floats yet.